echion/frame.h

```diff
@@ -450,50 +455,82 @@ Frame &Frame::read(PyObject *frame_addr, PyObject **prev_addr)
 
 #if PY_VERSION_HEX >= 0x030b0000
 // ------------------------------------------------------------------------
-Frame &Frame::read_local(_PyInterpreterFrame *frame_addr, PyObject **prev_addr)
+Frame &Frame::read_with_stack_chunk(_PyInterpreterFrame *frame_addr, PyObject **prev_addr)
```

Comment:
This function can actually be removed by merging its body to `read()` function above using preprocessor macros